supervisely/geometry/mask_3d.py

```diff
@@ -483,3 +522,49 @@ def base64_2_data(encoded_string: str) -> np.ndarray:
             data = data.reshape(shape)
             logger.debug("Converted successfully!")
         return data
+
+    def add_mask_2d(
+        self,
+        mask_2d: np.ndarray,
+        plane_name: Literal["axial", "sagittal", "coronal"],
```

Comment:
literal with Plane.AXIAL

Reply:
circular import if we import `Plane` and Plane.AXIAL can't be used as Litera[Plane.AXIAL]